main.go

```diff
@@ -16,15 +16,16 @@ limitations under the License.
 package main
 
 import (
-	"github.com/drew-viles/baskio/cmd"
 	"log"
 	"os"
 	"runtime"
+
+	"github.com/drew-viles/baskio/cmd"
```

Comment:
I thought I'd already changed all of this for the eschercloud project so I'll get this done in the next release.

Reply:
I've created an issue for this so I don't forget to get it sorted